content/guides/core/automations/create-automations/webhook.md

```diff
@@ -0,0 +1,305 @@
+1. If the webhook requires a password or other sensitive string in its payload, set **Secret** to the secret that contains it. When you configure the automation that uses the webhook, you can access the secret as a [payload variable]({{< relref "#payload-variables" >}}) by prefixing its name with `$`.
+
+    If the webhook's access token is stored in a secret, you must _also_ complete the next step to specify the secret as the access token.
+1. To verify that the W&B can connect and authenticate to the endpoint:
+    1. Optionally, provide a payload to test. To refer to a secret the webhook has access to in the payload, prefix its name with `$`. This payload is only used for testing and is not saved. You configure an automation's payload when you [create the automation]({{< relref "#create-a-webhook-automation" >}}). See [Troubleshoot your webhook]({{< relref "#troubleshoot-your-webhook" >}}) to view where the secret and access token are specified in the `POST` request.
+    1. Click **Test**. W&B attempts to connect to the webhook's endpoint using the credentials you configured. If you provided a payload, W&B sends it.
+
+    If the test does not succeed, verify the webhook's configuration and try again. If necessary, refer to [Troubleshoot your webhook]({{< relref "#troubleshoot-your-webhook" >}}).
+
+Now you can [create an automation]({{< relref "#create-a-webhook-automation" >}}) that uses the webhook.
+
+## Create an automation
+After you [configure a webhook]({{< relref "#reate-a-webhook" >}}), select **Registry** or **Project**, then follow these steps to create an automation that triggers the webhook.
+
+{{< tabpane text=true >}}
+{{% tab "Registry" %}}
+A Registry admin can create automations in that registry. Registry automations are applied to all collections in the registry, including those added in the future.
+
+1. Log in to W&B.
+1. Click the name of a registry to view its details, 
+1. To create an automation scoped to the registry, click the **Automations** tab, then click **Create automation**. An automation that is scoped to a registry is automatically applied to all of its collections (including those created in the future).
+
+    To create an automation scoped only to a specific collection in the registry, click the collection's action `...` menu, then click **Create automation**. Alternatively, while viewing a collection, create an automation for it using the **Create automation** button in the **Automations** section of the collection's details page.
+1. Choose the [**Event**]({{< relref "/guides/core/automations/automation-events.md" >}}) to watch for.
+
+    1. Fill in any additional fields that appear, which depend upon the event. For example, if you select **An artifact alias is added**, you must specify the **Alias regex**.
+    
+    Click **Next step**.
```

Comment:
This is formatted oddly in the preview docs...I think the inlines are is messing up the desired format?